models/prices/bnb/prices_bnb_tokens.sql

```diff
@@ -198,5 +198,6 @@ FROM
     ("stg-stargatetoken","bnb","STG","0xb0d502e938ed5f4df2e681fe6e419ff29631d62b",18),
     ("ico-axelar", "bnb", "AXL", "0x8b1f4432f943c465a973fedc6d7aa50fc96f1f65", 6),
     ("lvl-level","bnb","LVL","0xb64e280e9d1b5dbec4accedb2257a87b400db149",18)
-    
+    ("lvl-level","bnb","GHNY","0xa045E37a0D1dd3A45fefb8803D22457abc0A728a",18)
+        
```

Comment:
Hi @roflfuso, thank you for the PR.

To add `GHNY` into prices.usd table, firstly, we need to verify whether the address is correct using explorer:
https://bscscan.com/token/0xa045E37a0D1dd3A45fefb8803D22457abc0A728a

Then, we need to make sure its token-id is available and active on coinspaprika:
https://api.coinpaprika.com/v1/coins/ghny-grizzly-honey

Next, we can construct the line:
("ghny-grizzly-honey","bnb","GHNY","0xa045e37a0d1dd3a45fefb8803d22457abc0a728a",18)

And we need to add `,` before this line to make syntax lgtm.

I will update this line in the next commit.